src/styles/card/image.scss

```diff
@@ -0,0 +1,9 @@
+.images {
+  background-image: url(../images/imac.jpeg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 3px;
```

Comment:
The margin property is not usually used with images. If you need to position the image, it's better to use padding in the parent element. Also, it's not clear why you need the left and right margins to be 3px. If you need to center the image, there are better ways to do it, for example, by using flexbox in the parent element.